src/constants.js

```diff
@@ -31,7 +31,8 @@ export const UNIQUE_OPTIONS_MAX = 50;
 
 export const saveMessages = {
   success: "Your model was saved!",
-  failure: "There was an error. Your model did not save. Please try again."
+  failure: "There was an error. Your model did not save. Please try again.",
+  piiProfanity: "Your model could not be saved because it contains profanity or personally identifying information (e.g. email, address, phone number, etc)."
```

Comment:
nit: Apparently there is no need to have "etc." when starting with "e.g.", because it's a list of examples, rather than being exhaustive.